cocos/core/animation/cross-fade.ts

```diff
@@ -110,6 +125,10 @@ export class CrossFade extends Playable {
             easeTime: 0,
             target,
         });
+
+        if (!this.isMotionless) {
```

Comment:
A new cross fade request cause `CrossFade` reshedule itself, if it has not been paused or stopped; if so, it'd be sheduled once got playing or resumed.